fastcrypto-zkp/src/bn254/zk_login_api.rs

```diff
@@ -162,7 +162,7 @@ pub fn verify_zk_login(
 }
 
 /// Verify a proof against its public inputs using the fixed verifying key.
-fn verify_zk_login_proof_with_fixed_vk(
+pub fn verify_zk_login_proof_with_fixed_vk(
```

Comment:
This is not ideal but necessary to be able to benchmark it. I think this is better than adding a new feature flag for now.